src/preferences/types.ts

```diff
@@ -16,6 +16,7 @@ export type UserPreferences = {
   overrideColorScheme?: boolean;
   language?: Preference_Language;
   useSystemLanguage?: boolean;
+  defaultUserTypeString?: string;
```

Comment:
Having a hard time understanding what user type and profile is until reading the translation files. Are these names too ambigues?

Reply:
Yeah, we thought about using our own domain names for these things. There even is an iteration of the code where the name `TravellerType` is used instead of `UserProfile`. But after some contemplation we decided on reusing the domain types and names from Entur, which is `UserProfile`. And `UserProfile` has a field `userTypeString` which we now store as the default value in the preferences.